roles/openshift-management/tasks/prune-projects.yml

```diff
@@ -0,0 +1,39 @@
+---
+- name: Set username fact (if this is run by an Ansible Tower job)
+  set_fact:
+    run_as: "--as='{{ tower_user_name }}'"
+  when: 
+  - tower_user_name is defined
+  - tower_user_name|trim != ''
+
+- name: Get list of projects
+  shell: oc {{ run_as | default('') }} get project --no-headers | awk '{print $1}'
+  environment:
+    KUBECONFIG: "{{ kubeconfig }}"
+  register: projects 
+
+- name: Get OpenShift project timestamps
+  shell: oc {{ run_as | default('') }} get project {{ item }} --template={{ '{{' }}.metadata.creationTimestamp{{ '}}' }}
+  environment:
+    KUBECONFIG: "{{ kubeconfig }}"
+  register: project_timestamp
+  with_items: "{{ projects.stdout_lines }}"
```

Comment:
Had to replace with this line, otherwise an error was thrown in Ansible 2.2 when pruning projects was disabled

```
with_items: "{{ projects.stdout_lines | default([]) }}"
```